snooty/diagnostics.py

```diff
@@ -149,6 +149,21 @@ def __init__(
         self.name = name
 
 
+class UnexpectedNodeType(Diagnostic):
+    severity = Diagnostic.Level.error
+
+    def __init__(
+        self,
+        found_type: Union[str, None],
+        expected_type: Optional[str],
+        start: Union[int, Tuple[int, int]],
+    ) -> None:
+        suggestion = f' Expected: "{expected_type}".' if expected_type else ""
```

Comment:
Leading space intentional?

Reply:
It was, but I went ahead and changed it to hopefully make it less confusing